Feature/kas 4433 send to vp role permissions

sergiofenoll opened the pull request:
https://kanselarij.atlassian.net/browse/KAS-4433

"Doorsturen naar het Vlaams Parlement" button is now shown to Admin (& secretarie & kanselarij) & Kabinetdossierbeheerders profiles. Other profiles can see the sending to VP (with the green banner) but cannot send to VP.

Related PRs:
- [ ] https://github.com/kanselarij-vlaanderen/app-kaleidos/pull/439
- [ ] https://github.com/kanselarij-vlaanderen/cache-warmup-service/pull/14

sergiofenoll committed:
feat: move manage-agendaitems check inside agendaitem-controls component

This lets us show the component to all profiles, not only those who can
manage the agendaitems (specifically for sending to VP button)

sergiofenoll committed:
feat: expand canSendToVP check to take into account dossierbeheerders

tdn commented on app/config/permissions.js:
```diff
@@ -110,6 +113,7 @@ const groups = [
       'view-decisions-before-release',
       'search-confidential-cases',
       'search-confidential-documents',
+      'send-cases-to-vp',
```
I was under the impression that secretarie/kanselarij would never send these, unless that changed while I was on vacation?

tdn commented on app/config/permissions.js:
```diff
@@ -138,6 +142,7 @@ const groups = [
       'view-decisions-before-release',
       'search-confidential-cases',
       'search-confidentnial-documents',
+      'send-cases-to-vp',
```
I was under the impression that secretarie/kanselarij would never send these, unless that changed while I was on vacation?

tdn commented on app/config/permissions.js:
```diff
@@ -168,6 +173,7 @@ const groups = [
       'view-decisions-before-release',
       'search-confidential-cases',
       'search-confidential-documents',
+      'send-cases-to-vp',
```
OVRB can send cases to VP? That doesn't seem right to me...

tdn reviewed (changes requested):
We need to double check during/after the standup tomorrow whether these permissions are correct, because they are different than I remember putting into the story...

sergiofenoll committed:
chore: remove send to VP permission from secretarie/KB

sergiofenoll committed:
fix: don't try to do VP checks if there's no subcase

This is the case in the "approval for meeting" agendaitem, which every
agenda has and otherwise causes errors.
